components/Switch/Switch.themes.ts

```diff
@@ -0,0 +1,22 @@
+import tinycolor from 'tinycolor2';
+import { Property } from '@stitches/react/types/css';
+export namespace Theme {
+  type Colors = {
```

Comment:
type is missing `switchActiveBackground`

Reply:
fixed, I also updated from upstram